README.md

```diff
@@ -374,21 +394,35 @@ Changing the `listen` or `upstream` fields will restart the proxy and drop any a
 If `listen` is specified with a port of 0, toxiproxy will pick an ephemeral port. The `listen` field
 in the response will be updated with the actual port.
 
-If you change `enabled` to `false`, it'll take down the proxy. You can switch it
+If you change `enabled` to `false`, it will take down the proxy. You can switch it
 back to `true` to reenable it.
 
+#### Toxic fields:
```

Comment:
Clarify that these are the fields on all toxics (global) and that a toxic may extend the struct with additional.